en/docs/apis/api-concepts.md

```diff
@@ -0,0 +1,33 @@
+# Concepts
+
+Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+ 
+## API
+
+An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+
```

Comment:
Remove extra new lines. Check all the places

Reply:
Removed extra new lines.